helm-charts/dnsbl-exporter/values.yaml

```diff
@@ -0,0 +1,339 @@
+
+## Configure extra options for Application containers' liveness, readiness and startup probes
+## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/#configure-probes
+## @param startupProbe.enabled Enable startupProbe
+## @param startupProbe.initialDelaySeconds Initial delay seconds for startupProbe
+## @param startupProbe.periodSeconds Period seconds for startupProbe
+## @param startupProbe.timeoutSeconds Timeout seconds for startupProbe
+## @param startupProbe.failureThreshold Failure threshold for startupProbe
+## @param startupProbe.successThreshold Success threshold for startupProbe
+##
+startupProbe:
+  enabled: false
+  initialDelaySeconds: 120
+  periodSeconds: 10
+  timeoutSeconds: 5
+  failureThreshold: 6
+  successThreshold: 1
+## @param livenessProbe.enabled Enable livenessProbe
+## @param livenessProbe.initialDelaySeconds Initial delay seconds for livenessProbe
+## @param livenessProbe.periodSeconds Period seconds for livenessProbe
+## @param livenessProbe.timeoutSeconds Timeout seconds for livenessProbe
+## @param livenessProbe.failureThreshold Failure threshold for livenessProbe
+## @param livenessProbe.successThreshold Success threshold for livenessProbe
+##
+livenessProbe:
+  enabled: true
+  initialDelaySeconds: 120
+  periodSeconds: 10
+  timeoutSeconds: 5
+  failureThreshold: 6
+  successThreshold: 1
+## @param readinessProbe.enabled Enable readinessProbe
+## @param readinessProbe.initialDelaySeconds Initial delay seconds for readinessProbe
+## @param readinessProbe.periodSeconds Period seconds for readinessProbe
+## @param readinessProbe.timeoutSeconds Timeout seconds for readinessProbe
+## @param readinessProbe.failureThreshold Failure threshold for readinessProbe
+## @param readinessProbe.successThreshold Success threshold for readinessProbe
+##
+readinessProbe:
+  enabled: true
+  initialDelaySeconds: 30
+  periodSeconds: 5
+  timeoutSeconds: 3
+  failureThreshold: 6
+  successThreshold: 1
+## @param customLivenessProbe Custom livenessProbe that overrides the default one
+##
+customLivenessProbe: {}
+## @param customReadinessProbe Custom readinessProbe that overrides the default one
+#
+customReadinessProbe: {}
+## Custom startupProbe block
+customStartupProbe: {}
+
+image:
+  registry: ghcr.io
+  repository: luzilla/dnsbl_exporter
+  tag: v0.6.0
+  pullPolicy: IfNotPresent
+  ## Optionally specify an array of imagePullSecrets.
+  ## Secrets must be manually created in the namespace.
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  ## e.g:
+  ## pullSecrets:
+  ##   - myRegistryKeySecretName
+  ##
+  pullSecrets: []
+
+configuration:
+  rbls:
+    # 0SPAM
+    - bl.0spam.org
+    - rbl.0spam.org
+    # NIXSPAM
+    - ix.dnsbl.manitu.net
+    # Spamhaus Zen
+    - zen.spamhaus.org
+  targets:
+    - smtp.fastmail.com
+  resolver: "1.1.1.1:53"
+
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
```

Comment:
Wondering if this config should be removed, along with the templating for it. As in, does it need a service account to run? I think no? But let me know.

Reply:
re: `nameOverride`/`fullnameOverride` - I rely on Bitnami's template, but if you do not like it's output or, probably, want to deploy the same chart twice into the same namespace - that might be helpful.

re: `serviceAccount`: every container has the permissions in the cluster which are being managed also through Service Accounts. If no service account is specified, the default would be used. Service accounts could be also used to talk to APIs (like Kubernetes itself, like AWS, Azure, so on...). At AWS Elastic Kubernetes Service (EKS), saying, in order to talk to AWS API service account has a special annotation with IAM Role arn.

In the case when we do not specify anything the account with the default permissions would be created. If the Kubernetes cluster (or policy, etc) requires the certain SA (service account) to be used (it could be pre-created by kubernetes cluster administrator) - we can specify its name and use the pre-created account.

Reply:
@zentavr Can you look into `automountServiceAccountToken: false`?